### ISSUE TITLE
Unable to Publish Tournament if Some Changes Made

### ISSUE THREAD

**Issue by NikAlexander** [BattlePro/battlepro-issues#55]
**Issue:**
Error report produced when attempting to update maximum number of teams before any players have signed up.

**Repro:**
1. Create new tournament and set max number of teams to join as 32
2. Publish the tournament and then click 'manage'
3. Select 'Game Info' and update max teams from 32 to 64 and click 'save'
![maxteam](https://user-images.githubusercontent.com/6005869/29753535-80b4231a-8b41-11e7-9271-a923ff651ed5.JPG)
4. After updated settings are saved, click 'Publish' to publicly update
![publish](https://user-images.githubusercontent.com/6005869/29753742-ec85461a-8b45-11e7-89ed-5f763d78f372.JPG)
5. After clicking 'Publish', observe error report
![error](https://user-images.githubusercontent.com/6005869/29753541-9b1b850e-8b41-11e7-9b97-4a13d7892f54.JPG)
6. Notice that the tournament is NOT published or is NOT visible on the Halo Tournaments listing page.  If it is not visible, users are not able to sign up.
![not published](https://user-images.githubusercontent.com/6005869/29753759-532dc8ba-8b46-11e7-9a32-5caea49361e0.JPG)

**UPDATE:**
- If changes are made to 'Require Wager' and then 'Published' with updated prizing information, the tournament will get put into this error mode as well.  Seems if any structural changes are made to the tournament, it will go into an 'error' mode and will not be able to be resolved.

**Comment by NikAlexander** [comment]
**NOTE:**
When in 'manage' mode and making changes to tournament details text, it changed the listing from being 'published' to 'unpublished' without notification.  When attempting to publish, it produced an error report saying that 'staging has not been selected', which is false.  Staging was already selected however I needed to click 'save' on that page for it to recognize those selections.  Only after I saved 'staging' page was I able to successfully 'publish' the tournament for public view.
